### DISCUSSION TITLE
Is there a model to remove dual vocal/singer (male and female duet singer)?

### DISCUSSION THREAD

**Discussion by RAG23** [Anjok07/ultimatevocalremovergui#787] [Ideas]
im trying to seperate both male and female singer from the song "love is an open door" is that even possible?

**Comment by deadniell** [comment]
You mean when those both male and female voices sing along? If yes, once you have your only vocals track, select that input and use VR Architecture 5_HP or 6_HP-KARAOKE-UVR.
- Instrumental only will do the "background" voice as in for example the male voice
- vocals only will do the main voice as in the female voice
